pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -54,6 +55,8 @@ public class ConvertResourceFn extends FetchSearchPageFn<HapiRowDescriptor> {
 
   private final Boolean processDeletedRecords;
 
+  Counter counter = Metrics.counter(MetricsConstants.METRICS_NAMESPACE, MetricsConstants.DATA_FORMAT_EXCEPTION_ERROR);
```

Comment:
I prefer if we follow the same pattern as other counters' initialization but we can leave that for a future PR. I think the initialization might need to be moved out of the constructor too (I'll address these later).

Reply:
Resolving this for now to merge.